src/main/java/Main.java

```diff
@@ -1,9 +1,17 @@
+import java.util.Scanner;
+
 /**
  * Created by iyasuwatts on 10/17/17.
  */
 public class Main {
 
     public static void main(String[] args ){
 
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter your name: ");
+
+        String userName = sc.next();
+        System.out.println("Hello, " + userName + "!");
```

Comment:
you need to check for "Bob" or "Alice", the code does not implement the requirements.